cypress/e2e/buy-tshirt.cy.ts

```diff
@@ -0,0 +1,54 @@
+import {
+  MenuContentPage,
+  LoginPage,
+  AddressStepPage,
+  ShippingStepPage,
+  PaymentStepPage,
+  ShopingCartPage,
+  ProductListPage,
+} from "../page/index";
+
+const menuContentPage = new MenuContentPage();
+const loginPage = new LoginPage();
+const addressStep = new AddressStepPage();
+const shippingStep = new ShippingStepPage();
+const paymentStep = new PaymentStepPage();
+const shopingCartPage = new ShopingCartPage();
+const productListPage = new ProductListPage();
```

Comment:
remove these lines, the idea is to declare them within the describe as `let` (which you are already doing)